docs/contributing/code-style.md

```diff
@@ -71,13 +64,20 @@ The Vagrant setup process runs this for you.
   [isort](https://pycqa.github.io/isort/))
 - templates
 - Puppet configuration
-- custom checks (e.g. trailing whitespace and spaces-not-tabs)
+- custom checks (e.g., trailing whitespace and spaces-not-tabs)
+
+If you'd like, you can set up a local Git commit hook that will lint
+only your changed files each time you commit:
```

Comment:
I think this section should somehow advertise the ability to run these against individual target files -- it's documented in detail on the page on linters, but a few words on that being a good idea would be valuable here.